<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Data</class>
 <widget class="QDialog" name="Dialog_Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Details</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>215</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>238</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_ShoeData">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="verticalSpacing">
     <number>7</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Brand">
      <property name="text">
       <string>Marke</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Brand"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Model">
      <property name="text">
       <string>Modell</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Model"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_CW">
      <property name="text">
       <string>Colorway</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_CW"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Modelnr">
      <property name="text">
       <string>Modellnummer</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Modelnr"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_Releasedate">
      <property name="text">
       <string>Releasedatum</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="dateEdit_Releasedate">
      <property name="date">
       <date>
        <year>2017</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_Buydate">
      <property name="text">
       <string>Kaufdatum</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDateEdit" name="dateEdit_Buydate">
      <property name="date">
       <date>
        <year>2017</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_Price">
      <property name="text">
       <string>Kaufpreis</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_Price">
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_Seller">
      <property name="text">
       <string>Verkäufer</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_Seller"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_Size">
      <property name="text">
       <string>Größe</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_Size">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>4.000000000000000</double>
      </property>
      <property name="maximum">
       <double>16.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>10.500000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>161</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_BrowsePics">
    <item row="0" column="0">
     <widget class="QLabel" name="label_BrowsePic1">
      <property name="text">
       <string>Bild 1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_BrowsePic1">
      <property name="text">
       <string>Durchsuchen...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_BrowsePic2">
      <property name="text">
       <string>Bild 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_BrowsePic2">
      <property name="text">
       <string>Durchsuchen...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_PreviewPic1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>9</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_PreviewPic2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_Data</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_Data</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
